config/locales/interface/output_elements/en_merit.yml

```
@@ -123,6 +123,20 @@ en:
         demand.\r\n<br/><br/>\r\nThe table shows what place in the merit order each
         technology has, which is based on the operating costs. Also the table shows
         the available capacity of each technology as well as the full load hours."
+    flexibility_monthly_carriers_inflexible_supply_demand:
+      title: 1. Supply and demand per month for E, G, H<sub>2</sub> en H
```

Personally, I'm not a fan of the numbers being part of the chart titles; they won't mean anything to most visitors. To match the changes @michieldenhaan made to the Dutch translations

```suggestion
      title: Supply and demand per month
```